client/scss/components/_dialog.scss

```diff
@@ -157,6 +157,13 @@
   .w-dialog__box {
     pointer-events: auto;
     box-shadow: theme('boxShadow.md');
+    overflow: hidden;
+    border-top-left-radius: 10px;
+    border-top-right-radius: 10px;
+  }
+  .w-dialog__message{
+    border-top-left-radius: 10px;
+    border-top-right-radius: 10px;
```

Comment:
If we're going with restoring the `overflow`, these aren't necessary

Reply:
I'd like us to try to solve this without adding overflow hidden if possible, that tends to cause other bugs down the road.